tests/unit/lms/services/jwt_test.py

```diff
@@ -182,6 +198,14 @@ def svc(self, lti_registration_service, rsa_key_service):
         svc._get_jwk_client.cache_clear()  # pylint: disable=protected-access
         return svc
 
+    @pytest.fixture
+    def json(self, patch):
+        return patch("lms.services.jwt.json")
+
+    @pytest.fixture
+    def jwe(self, patch):
+        return patch("lms.services.jwt.jwe")
```

Comment:
Why are we mocking these? Why not call the real `json` and `jwe` libs? It's better not to mock standard library or third-party dependencies unless we have to (e.g. if they're going to send network requests or something). I guess if you're not mocking it then for a test assertion the test would have to decrypt and assert that it gets back the contents that it put in.

Reply:
This is an artifact of the approach I took here. This is copy/pasted code from h-vialib which has the methods for de/encrypt. The methods are individually tested with the style from this PR and there's one other test that test the full encrypt/decrypt cycle without mocks.

I reckon we should fix this but I'm going to merge this and add a note to the board about it. Maybe an h-security lib is not a bad idea after all.